src/js/components/Skeleton/__tests__/__snapshots__/Skeleton-test.tsx.snap

```diff
@@ -634,12 +646,6 @@ exports[`Skeleton with styling props 1`] = `
   -webkit-font-smoothing: antialiased;
 }
 
-@media only screen and (max-width: 768px) {
```

Comment:
this went away because of this change correct?

```
 ${(props) => props.round && roundStyle(props.round, true, props.theme)}
  ${(props) =>
    props.round && roundStyle(props.round, props.responsive, props.theme)}
```

Reply:
Yes. It wasn't passing along the responsive like it really should.